src/js/media-controller.js

```diff
@@ -43,7 +44,7 @@ const DEFAULT_TIME = 0;
  */
 class MediaController extends MediaContainer {
   static get observedAttributes() {
-    return super.observedAttributes.concat('nohotkeys', 'hotkeys');
+    return super.observedAttributes.concat('nohotkeys', 'hotkeys', 'default-stream-type');
```

Comment:
Question: Should this have dashes or not? I see hotkeys/nohotkeys didn't follow the dash format (more HTML-like). Do we want to move that direction in this area where it's not a media-state attribute?

Reply:
So right now the not quite perfect distinction we have is:

1. media state attributes: attribute names are hyphenated
2. Media Chrome component-specific attributes: unhyphenated

This one falls right in the middle, but I'd lean towards treating it like (1) since it directly relates to defaulting `media-stream-type`.

Reply:
fwiw, I'm definitely on board with dropping hyphens on everything (relates to forward looking framework compat/ease when working with web components), though @dylanjha has some strong gut against readability/devex for "smashed attribute names". I just think that should be an "all at once" decision, if we make the change.